packages/pubsub/lib/utils/serde.ts

```diff
@@ -0,0 +1,23 @@
+import { serialize, deserialize } from 'v8';
+import type { Result } from '@nangohq/utils';
+import { Ok, Err } from '@nangohq/utils';
+
+export const serde = {
```

Comment:
I guess it means serialize/deserialize? not super clear but it seems to be used :p

Reply:
yes, I thought that was super common. Maybe I have been doing too much rust 😆